workspaces/libnpmexec/lib/run-script.js

```diff
@@ -31,7 +32,9 @@ const run = async ({
     },
   }
 
-  npmlog.disableProgress()
+  if (stdio === 'inherit') {
+    npmlog.disableProgress()
```

Comment:
This is done so that the output from npm doesn't interlace with the output of your script.  We always want to disable progress bars during exec.